contractsSol5/Dao/KyberFeeHandler.sol

```diff
@@ -196,21 +201,26 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils4, BurnConfigPermission {
             return true;
         }
 
+        WeiData memory weiData;
+        uint epoch;
         // Decoding BRR data
-        (uint rewardWei, uint rebateWei, uint epoch) = getRRWeiValues(feeBRR);
+        (weiData.reward, weiData.rebate, epoch) = getRRWeiValues(feeBRR);
 
+        uint totalRebateBps;
         for (uint i = 0; i < rebateWallets.length; i++) {
             // Internal accounting for rebates per reserve wallet (rebatePerWallet)
-            rebatePerWallet[rebateWallets[i]] += rebateWei * rebateBpsPerWallet[i] / BPS;
+            rebatePerWallet[rebateWallets[i]] += weiData.rebate * rebateBpsPerWallet[i] / BPS;
+            totalRebateBps += rebateBpsPerWallet[i];
         }
+        require(totalRebateBps <= BPS);
```

Comment:
Please add revert message here
Do we have check for `rebateBpsPerWallet[i] <= BPS` anywhere?